.github/workflows/codeql.yml

```diff
@@ -26,7 +26,7 @@ jobs:
 
   analyze:
     name: Analyze
```

Comment:
Any reason not to keep the ubuntu-latest?

Reply:
Not married to it, just a slight preference for pinning the runner for the sake of transparency about the relationship between the runner version and check outcomes. 

That is, when the latest runner inevitably shifts to the next LTS, we may not notice if, by chance, there are no breaking changes, which isn't guaranteed.

If we pin the runner, we'll be forced to opt-in to the change when our pinned runner is eventually sunset, and it will be more obvious that bumping the pinner runner version is a factor in any check outcomes at that time.

There's a chance we'll never need to do anything if we use latest, but if we do have to fix something it'll be harder to figure out due to the ambiguity about which underlying running was working and whether it changed.